hyperbole-and-a-half.html

```diff
@@ -0,0 +1,71 @@
+<!Doctype>
+  <html>
+    <head>
+      <title>Hyberbole and a Half</title>
+    </head> 1
+  <body>
+    <header>
+      <h1>Hyperbole and a Half</h1>
+      <img src="images_hyperbole/Blogheader.png"> 
+    </header>
+      <nav>  
+        <h2>
+        <ul>
+          <li><a href="http://hyperboleandahalf.blogspot.com">Home</a></li>
+          <li><a href="http://hyperboleandahalf.blogspot.com/p/about.html">About</a></li>
+          <li><a href="http://hyperboleandahalf.blogspot.com/p/about-2.html">About #2</a></li>
+          <li><a href="http://hyperboleandahalf.blogspot.com/p/contact.html">Contact</a></li>
+          <li><a href="http://hyperboleandahalf.blogspot.com/p/faq_10.html">FAQ</a></li>
+          <li><a href="http://hyperboleandahalf.blogspot.com/p/store.html">STORE!</a></li>
+          <li><a href="http://hyperboleandahalf.blogspot.com/p/forum.html">Forum</a></li>
+        </ul>
+        </h2>
+      </nav>      
+    <article>
+      <section>
+
+      <h2> "The Alot is Better Than You at Everything"</h2>
+    
+        <p> "As a grammatically conscientious person who frequents internet forums and YouTube, I have found it necessary to develop a few coping mechanisms.  When someone types out "u" instead of "you," instead of getting mad, I imagine them having only one finger on each hand and then their actions seem reasonable.  If I only had one finger on each hand, I'd leave out unnecessary letters too!" </p>
+          <img src="images_hyperbole/ALOT10.png">    
```

Comment:
All your image tags should be on the same level as the p tags. The way you have them makes them look like they are children of the p tags rather than siblings.